### DISCUSSION TITLE
Looking to contribute

### DISCUSSION THREAD

**Discussion by Bibekkshetri055** [argoproj/argo-cd#11994] [Q&A]
I am a second year student looking to contribute to the open source . I am an active learner but I am stuck with how to get started in open source through this project. Can someone please guide me through this journey? I need your help.

**Comment by jaideepr97** [comment]
hi @Bibekkshetri055 thanks for your interest in the project! When I first started out I thought a good place to begin was first getting used to the project as a user 
once you're more familiar with it I would start with looking at issues lablelled 'good first issue' 
There's also a contributor office hours that happens every thursday around 11:15 am EST that you could start attending

**Reply by Bibekkshetri055** [reply]
Extremely thankful to your reply @jaideepr97 . I will surely attend those.
